common/reactive-grpc-common/src/main/java/com/salesforce/reactivegrpc/common/AbstractStreamObserverAndPublisher.java

```diff
@@ -126,6 +127,7 @@ public void request(long n) {
             Consumer<CallStreamObserver<?>> onSubscribe,
             Runnable onTerminate) {
         this.prefetch = prefetch;
+        this.limit = (prefetch * 2) / 3;
```

Comment:
Why multiply by 2/3? Maybe the factor should be a well named constant.

Reply:
Fixed. That factor specifies the number of elements after which the next prefetch should happen. It is better to have limit lower than prefetch (which is the number of elements requested at first) so the upstream will have some number of elements to send before prefetch is fulfilled by the upstream. It allows avoiding the case when the sender is blocked waiting for the next request.